File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int Height;
    int m = 1;
    do
    {
        Height = get_int("Height of the Pyramid?");
    }
    while (Height < 1 || Height > 8);
    int n = Height;
    for (int t = 1; t <= Height; t++)
    {
        for (int i = 1; i <= Height; i++)
        {
            if (i < n)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        printf("  ");
        for (int j = 1; j <= Height; j++)
        {
            if (j <= m)
            {
                printf("#");
            }
            else
            {
                continue;
            }
        }

        printf("\n");
        m = m + 1;
        n = n - 1;
    }
}

File: sentimental-mario-more/mario.py
Height = input("Height: ")
k = 0
while k < 1:
    try:
        Height = int(Height)
        k = 1
    except:
        Height = input("Height: ")
while Height > 8 or Height < 1:
    Height = input("Height: ")
    Height = int(Height)
j = 0
while j < Height:
    i = 1
    while i < Height - j:
        print(" ", end="")
        i = i + 1
    i = 0
    while i < j + 1:
        print("#", end="")
        i = i + 1
    print("  ", end="")
    i = 0
    while i < j + 1:
        print("#", end="")
        i = i + 1
    print("")
    j = j + 1
